userspace/libpman/src/programs.c

```diff
@@ -298,4 +299,31 @@ int pman_detach_signal_deliver()
 	return 0;
 }
 
+int pman_set_autoload_programs(const bool ppm_sc_of_interest[PPM_SC_MAX])
+{
+	char buff[256];
+
+	for (unsigned int i = 0; i < PPM_SC_MAX; i++) {
+		const char* name = scap_get_ppm_sc_name(i);
+		unsigned int name_len = strlen(name) < 4093 ? strlen(name) : 4093;
```

Comment:
this check needs to be adjusted for the new buff size

Reply:
🦀